internal/ioutil/tree.go

```diff
@@ -0,0 +1,58 @@
+package ioutil
+
+import (
+	"fmt"
+)
+
+const (
+	treeItemPrefix     = "├── "
+	treeItemPrefixLast = "└── "
+	subTreePrefix      = "│   "
+	subTreePrefixLast  = "    "
+)
+
+type Node struct {
+	Value    string
+	Children []*Node
+}
+
+func NewTree(value string) *Node {
+	node := Node{Value: value, Children: []*Node{}}
+	return &node
```

Comment:
Code can be simplified as
```suggestion
	return &Node{Value: value}
```